### DISCUSSION TITLE
lab6 the problem about run_vivado

### DISCUSSION THREAD

**Discussion by nthuyouwei** [bol-edu/HLS-SOC-Discussions#170] [Q&A]
各位好，我們在設計時，有多設計wbdecode以及原本4-1的bram，但在uart tcl檔案裏頭只有uart那部分，是否是要在vvd_srcs下加入自己的設計同時要修改tcl，想問除了下圖這部分要改，tcl檔案還有其他要改嗎?不確定是否會影響接線的部分。

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/2af13132-ff98-48cc-a47c-815d408a5b94)

**Comment by Lanxas1018** [comment]
同學好，tcl在rtl檔案include的部分需要在這兩處修改。
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/137324121/8047c468-47f1-42e6-983a-bd02536bebf0)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/137324121/e115ffea-7054-47e4-a69f-7c29405f1056)
